src/Juvix/Compiler/Concrete/Print/Base.hs

```diff
@@ -1014,11 +1014,18 @@ instance PrettyPrint PatternArg where
   ppCode PatternArg {..} = do
     let name' = ppCode <$> _patternArgName
         pat' = ppCode _patternArgPattern
-    (name' <&> (<> ppCode Kw.kwAt))
-      ?<> delimIf _patternArgIsImplicit delimCond pat'
-    where
-      delimCond :: Bool
-      delimCond = isJust _patternArgName && not (isAtomic _patternArgPattern)
+
+    let delimCond :: Bool
```

Comment:
These lets can be combined into a single let block.

Reply:
Done